mozilla/kuma
# Bug 1139490: Manage callout datauri with Pipeline

### Testing

1. Enter production mode
2. Verify that the homepage still uses a datauri for the callout box sprite
